src/stdlib/pascalcase.rs

```diff
@@ -0,0 +1,112 @@
+use crate::{compiler::prelude::*, stdlib::camelcase::into_case};
+
+use convert_case::{Case, Casing};
+
+use crate::stdlib::camelcase::variants;
```

Comment:
Could we move these definitions into a shared module that all of the functions depend on rather than have each function depend on `camelcase`? You could move these functions into a `casing` subdirectory of `src/stdlib` that has a `mod.rs`.

Reply:
sure